scaladoc/src/dotty/tools/scaladoc/SocialLinks.scala

```diff
@@ -19,5 +23,8 @@ object SocialLinks:
       case "gitter" => Left(errorPrefix + "For 'gitter' arg expected one argument: url")
       case "discord" if splitted.size == 2 => Right(Discord(splitted(1)))
       case "discord" => Left(errorPrefix + "For 'discord' arg expected one argument: url")
+      case LowercaseNamePattern() if splitted.size == 4 => Right(Custom(splitted(1), splitted(2), splitted(3)))
+      case LowercaseNamePattern() if splitted.size == 3 => Right(Custom(splitted(1), splitted(2), splitted(2)))
+      case LowercaseNamePattern() => Left(errorPrefix + "For 'custom' two minimum arguments are expected: url, white icon name, [dark icon name]")
```

Comment:
It should be `light` instead of `white`. Also, the sentence seems a bit off.
I suggest: `For the 'custom' link, a minimum of two arguments is expected: URL, light icon file name, [dark icon file name]`